Source/drlg_l1.cpp

```diff
@@ -2201,6 +2184,7 @@ void GenerateLevel(lvl_entry entry)
 	}
 
 	bool doneflag;
+	uint8_t floorID = 13;

```

Comment:
```suggestion
	constexpr uint8_t CatacombsFloorID = 13;
```
Last one. Recent changes for globals/constants have been moving to PascalCase as well, so trying to keep that convention where possible.